platform/macos/godot_application_delegate.h

```diff
@@ -35,18 +35,12 @@
 #import <AppKit/AppKit.h>
 #import <Foundation/Foundation.h>
 
-@interface GodotApplicationDelegate : NSObject <NSUserInterfaceItemSearching, NSApplicationDelegate> {
-	bool high_contrast;
-	bool reduce_motion;
-	bool reduce_transparency;
-	bool voice_over;
```

Comment:
Moved these to the `.mm` file, as they don't need to be in the public interface.